client/src/components/Form/Elements/FormDrilldown/utilities.ts

```diff
@@ -0,0 +1,24 @@
+export interface Option {
+    name: string;
+    value: string;
+    options: Array<Option>;
+}
+
+export type Value = string[] | string | null;
+
+export function getAllValues(headOptions: Array<Option>): string[] {
```

Comment:
Please consider annotating this function to improve code maintainability, as it's name and it's modules name are not very telling of it's purpose.